app/serializers/user_serializer.rb

```diff
@@ -78,7 +78,7 @@ def services
       BackendServices::HCA,
       BackendServices::EDUCATION_BENEFITS
     ]
-    service_list += BackendServices::MHV_BASED_SERVICES if object.loa3? && object.mhv_account_eligible?
+    service_list += BackendServices::MHV_BASED_SERVICES if object.mhv_account_eligible?
```

Comment:
@U-DON we may need to play around this, lets plan to do a test along side your branch tomorrow. I'm thinking this should maybe be object.mhv_account_eligible? && object.mhv_account.accessible?

because you should be able to use mhv_account state to determine whether or not the user is eligible for creation, right? Or am I confusing myself?

Reply:
nevermind, i think mhv_account_eligible on user should be good to go since it will return true for everything except "ineligible." In the future maybe this should be false unless "accessible?" and determining whether or not can create would live elsewhere on account creation controller or gated on mhv account state.